Articles/Blog/2025-02-coding-stanards-for-sustainable-software.md

```diff
@@ -1,6 +1,8 @@
 # Developing Coding Standards and Practices for Sustainable Software Development
 
-#### Contributed by [Manoj K. Bhardwaj](https://www.linkedin.com/in/manoj-bhardwaj-22448932/)
+#### Contributed by Manoj K. Bhardwaj
+
+<!-- Preferred profile link: https://www.linkedin.com/in/manoj-bhardwaj-22448932/ -->
```

Comment:
There does not seem to be any magic to pick this up and show at Manoj's profile page:

* https://preview.bssw.io/items?author=manoj-k-bhardwaj

(I regenerated the preview.bssw.io site to check.)

I will put this link in down in his bio.